### PR TITLE
Multiple sources

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: src/meta.rst [hunk 1]
```diff
@@ -95,3 +95,15 @@ Many packages use ``python setup.py install --single-version-externally-managed
 
 These options should be added to setup.py if a project uses setuptools. The goal is to prevent ``setuptools``
 from creating an ``egg-info`` directory because it does not interact well with conda.
+
+
+Downloading data files needed
```
Comment (ocefpaf) [inline src/meta.rst]: Maybe it should be: `Downloading extra sources and data files`

File: src/meta.rst [hunk 2]
```diff
@@ -95,3 +95,15 @@ Many packages use ``python setup.py install --single-version-externally-managed
...
+---------------------------------
+If you need additional data files for the build, download them using curl in the build script.
```
Comment (ocefpaf) [inline src/meta.rst]: ``If you need additional sources/data files``

Comment (ocefpaf) [inline src/meta.rst]: ``using curl in the build script and verify the checksum with openssl.``

File: src/meta.rst [hunk 3]
```diff
@@ -95,3 +95,15 @@ Many packages use ``python setup.py install --single-version-externally-managed
...
+Add curl and openssl to the build requirements and then you can use curl to download and openssl to verify.
+
+Example recipe is 
+`here <https://github.com/conda-forge/pari-feedstock/blob/187bb3bdd0a5e35b2ecaa73ed2ceddc4ca0c2f5a/recipe/build.sh#L27-L35>`_.
+
+Upstream issue for allowing multiple source is 
+`here <https://github.com/conda/conda-build/issues/1466>`_.
```
Comment (ocefpaf) [inline src/meta.rst]: :+1:

**Comment by ocefpaf** [comment]
Thanks @isuruf!

@kain88-de you asked this question on gitter. Do you have any suggestions for us here before we merge?

**Comment by kain88-de** [comment]
LGTM

**Pull request merged**